packages/gator-permissions-controller/src/GatorPermissionsController.ts

```diff
@@ -315,12 +386,15 @@ export default class GatorPermissionsController extends BaseController<
    *
    * @param args - The request parameters.
    * @param args.snapId - The ID of the Snap of the gator permissions provider snap.
+   * @param args.params - Optional parameters to pass to the snap method.
    * @returns A promise that resolves with the gator permissions.
    */
   async #handleSnapRequestToGatorPermissionsProvider({
     snapId,
+    params,
   }: {
     snapId: SnapId;
+    params?: Json;
```

Comment:
Should we have a more specific type for the params? I guess it's params for a specific method, so we should always know the type, and leaving it as Json is opening us up for bugs.